src/routes/oauthGuilds.js

```diff
@@ -11,10 +11,19 @@ module.exports = class extends Route {
 	}
 
 	async post(request, response) {
-		const botGuild = this.client.guilds.get(request.body.id);
-		const updated = await botGuild.settings.update(request.body.data, { action: 'overwrite' });
-		const errored = Boolean(updated.errors.length);
+		let botGuild, updated;
+		if (this.client.shard) {
```

Comment:
switch the blocks around, ur checking if shard exists that means u do the sharding calls in the 1st block not the 2nd